### PR TITLE
Modified for use with Rayfin cameras

### PR DESCRIPTION
**Pull request by krosburg**
Updated to be functional with new Rayfin cameras.

### CODE REVIEW

File: camds_agent.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+		self.ref_des = ref_des
+		self.share_name = share_name
+
+		self.img_dir_root = img_dir_root
+		self.image_dir = self.img_dir_root + self.ref_des
+		if not os.path.exists(self.image_dir):
+			os.makedirs(self.image_dir)
+
+		self.pending_file_queue = collections.deque('')
+		self.visited_dir_queue = collections.deque('') # kcr
```
Comment (oceanzus) [inline camds_agent.py]: What does this do?

Comment (krosburg) [reply camds_agent.py]: Woops. That last line is a debug line that I forgot to remove. I'm surprised that doesn't cause an error.

Comment (krosburg) [reply camds_agent.py]: Line 114 in the above comment was leftover debugging code and has removed

**Review by oceanzus — APPROVED** [review]
Looks okay.

**Pull request merged**